src/mermaid-isomorphic.ts

```diff
@@ -1,12 +1,14 @@
 import { type Mermaid, type MermaidConfig } from 'mermaid'
 import { type BrowserType, chromium, type LaunchOptions, type Page } from 'playwright'
 
-declare const mermaid: Mermaid
-
 const html = import.meta.resolve('../index.html')
-const mermaidScript = {
-  url: import.meta.resolve('mermaid/dist/mermaid.js')
-}
+
+const mermaidImportPath = new URL(import.meta.resolve('mermaid/dist/mermaid.esm.min.mjs')).pathname
```

Comment:
I prefer to use the unminified version. That way, if Mermaid throws an error, the stack trace is more useful.

I have my concerns about cross-platform compatibility when it comes to using paths. We may need to use file URLs as a string instead, which is what `import.meta.resolve()` returns anyway. I added a test to ensure cross-platform compatibility. Can you rebase?

Reply:
done in 32d9238158dd581ab16db049d4af13ad199f16f8